t/utils/batch-processor.t

```diff
@@ -316,7 +316,6 @@ GET /t
 done
 --- no_error_log
 BatchProcessor[log buffer] activating flush due to no activity
```

Comment:
`BatchProcessor ` -> `Batch Processor`?

We need to update the source code too.